service/src/main/resources/db/changesets/20220418_workspace_user_facing_id.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+- changeSet:
+    id: add workspace user_facing_id
+    author: melchang
+    changes:
+    - addColumn:
+        tableName: workspace
+        columns:
+          - column:
+              name: user_facing_id
+              type: text
+              constraints:
+                unique: true
+              remarks: |
+                A user-settable, mutable id. See PF-1359. Required (a later
```

Comment:
We might not want the PR numbers in the public record here. I don't know where this string goes though.

Reply:
There are a bunch of Jira ticket references in this repo...

Reply:
Yeah, the other concern was the longevity of this text, but it's associated with the transaction only, so it's fine.